src/Service.Tests/SqlTests/RestApiTests/Patch/MsSqlPatchApiTests.cs

```diff
@@ -190,6 +215,100 @@ public async Task PatchOneViewBadRequestTest()
             await base.PatchOneViewBadRequestTest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a rowversion field is missing from the request body.
```

Comment:
_successful execution_: no error thrown because `rowversion` was excluded from the request, right?

Reply:
right, clarified in the comment added.